app/src/main/res/drawable/feature_discussionform_bg_blue.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+
```

Comment:
Please write proper file names, there's a small mistake in this. It should be forum* instead of form I guess.